hack/tools/releasetags/releasetags.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"reflect"
+	"strings"
+
+	"github.com/blang/semver"
+	"sigs.k8s.io/kubebuilder/docs/book/utils/plugin"
+)
+
+type ReleaseTag struct{}
+
+// SupportsOutput checks if the given plugin supports the given output format.
+func (ReleaseTag) SupportsOutput(_ string) bool { return true }
+
+// Process modifies the book in the input, which gets returned as the result of the plugin.
+func (l ReleaseTag) Process(input *plugin.Input) error {
```

Comment:
I'm wondering if we could get some caching logic here: instead of querying a releasetag, check if it was queried before, if yes, use the cached value, if no, query it, and store the result in the cache.

I haven't checked the doc, so not sure if it's worth the effort. I'd be ok if you prefer to not do it now.

Reply:
I was thinking of a cache system too. But currently there only a few markers in the book. This is a good addition if more markers are added. So I think I won't implement it for now.